### ISSUE TITLE
[feature request] support `SHOW MEASUREMENTS where name =~ /.*paul.*/`

### ISSUE THREAD

**Issue by juddgaddie** [influxdb/influxdb#3523]
Since 0.9 is it no longer possible to search through measurements with a regex by the name of the measurement ?

Sorry if this has been answered already.

**Comment by mjdesa** [comment]
@juddgaddie currently you cant filter measurements with a regex.

**Comment by mjdesa** [comment]
Docs issue fixed by https://github.com/influxdb/influxdb.com/commit/be70610deaa39f35bb5a3fe1b09e24928ac6443b

**Comment by juddgaddie** [comment]
Are there any plans to introduce filtering on measurements ?

**Comment by mjdesa** [comment]
yes

**Comment by bubenkoff** [comment]
@mjdesa is there an issue opened already so we can subscribe?

**Comment by mjdesa** [comment]
@bubenkoff I thought there was, but I cant seem to find it. I'll reopen this ticket and relabel it as a feature request.

**Comment by bubenkoff** [comment]
thanks!

**Comment by pauldix** [comment]
@bubenkoff how many unique measurements do you have? Is there a reason you aren't using tags?

**Comment by bubenkoff** [comment]
thousands, but i agree, it's a bit strange to have that many
we're now reducing that number to hundreds

**Comment by dgnorton** [comment]
`WHERE name =~ /.*paul.*/` will prevent `name` from being used as a tag key.  Is there an alternative syntax we could use?  Maybe...

```
SHOW MEASUREMENTS FROM /.*paul*./ WHERE name = 'foo'
```
or
```
SHOW MEASUREMENTS WITH NAME =~ /.*paul*./ WHERE name = 'foo'
```

**Comment by gunnaraasen** [comment]
`SHOW MEASUREMENTS WITH NAME =~ /.*paul*./` would fit the `SHOW TAG VALUES` statement, e.g. `SHOW TAG VALUES FROM database WITH KEY = 'something'`.

Another alternative would be the MySQL style `LIKE` for regexes.

```
SHOW MEASUREMENTS FROM database WITH NAME LIKE /.*/ WHERE tagkey = 'tagvalue'
```

**Comment by beckettsean** [comment]
Replied on the duplicate ticket, but what about

`SHOW MEASUREMENTS WHERE MEASUREMENT =~ /.*/`

**Comment by dgnorton** [comment]
Implemented by #4501
